src/System.Private.CoreLib/src/System/Reflection/Assembly.cs

```diff
@@ -130,6 +136,12 @@ public override string ToString()
                 return displayName;
         }
 
+        /*
+          Returns true if the assembly was loaded from the global assembly cache.
+        */
+        public virtual bool GlobalAssemblyCache { get { throw NotImplemented.ByDesign; } }
```

Comment:
These should be also overrides for these two on RuntimeAssembly that return false and 0.